src/index.ts

```diff
@@ -82,97 +46,25 @@ export const getRequired = (details: Describe, settings: Settings): boolean => {
   }
 };
 
-export const getPropertiesAndInterfaces = (details: Describe, settings: Settings): PropertiesAndInterfaces => {
-  const result: PropertiesAndInterfaces = { properties: [], interfaces: [] };
-
-  if (!details.keys) {
-    return result;
-  }
-
-  for (const [name, key] of Object.entries(details.keys)) {
-    const propertyObject = key as Describe;
-
-    const type = getPropertyType(propertyObject);
-    if (!type) {
-      if (settings.debug) {
-        console.log('Property Type not found');
-      }
-      continue;
-    }
-
-    const description = propertyObject.flags?.description;
-
-    const required = getRequired(propertyObject, settings);
-
-    // only show description if it has one, no need to double up
-    const propertyJsDoc = `  /**
-   * ${description ? description : name}
-   */`;
-
-    const content = `${propertyJsDoc}
-  ${name}${required ? '' : '?'}: ${type.typeName};`;
-    const property: Property = {
-      name,
-      type: type.typeName,
-      content,
-      customType: isTypeCustom(type.baseTypeName) ? type.baseTypeName : undefined
-    };
-
-    result.properties.push(property);
-  }
-
-  // Sort Properties
-  result.properties = result.properties.sort((property1, property2) => 0 - (property1.name > property2.name ? -1 : 1));
-
-  return result;
-};
-
-export const convertSchema = (settings: Settings, joi: AnySchema): InterfaceRecord[] => {
-  const types: InterfaceRecord[] = [];
-
+export const convertSchema = (settings: Settings, joi: AnySchema): InterfaceRecord | undefined => {

```

Comment:
this is cleaner

Reply:
I can rename it, i think `ParsedSchema` makes sense

Reply:
I ended up going with `ConvertedType` since it was already a ts string at this point